tbx/static_src/sass/components/_call-to-action.scss

```diff
@@ -0,0 +1,29 @@
+@use "config" as *;
+
+.call-to-action {
+    background-color: var(--color--streamfield-background);
+    padding: $spacer-small;
+
+    @include media-query(large) {
+        padding: $spacer-medium;
+    }
+
+    &__inner {
+        @include media-query(large) {
+            display: flex;
+            align-items: flex-end;
+            justify-content: space-between;
+            gap: 40px;
+        }
+    }
+
+    &__text {
+        @include font-size(heading-four);
+        color: var(--color--heading);
+        max-width: 720px;
```

Comment:
Query: is this max-width used for other streamfields anywhere / related to any other widths? It looks like it needs to align on the right with the paragraph content above - just wondering if a variable can be used?

Reply:
The other streamfields widths are controlled by the grid but this component is unique in that it extends to the edge of the screen. There may be more instances so i'll assign it to a variable - more chance other devs will follow suit then : )

Reply:
https://github.com/torchbox/torchbox.com/pull/36#:~:text=to%20a%20var-,e0f7dd0,-Merge%20state